docs/quickstart/junit_4_quickstart.md

```diff
@@ -0,0 +1,77 @@
+# JUnit 4 Quickstart
+
+It's easy to add Testcontainers to your project - let's walk through a quick example to see how.
+
+Let's imagine we have a simple program that has a dependency on Redis, and we want to add some tests for it.
+ 
+You can see the implementation code and the corresponding test before we've added Testcontainers below:
+
+!!! example "Scenario code"
+    <!--codeinclude-->
+    [Implementation](../examples/src/main/java/quickstart/RedisBackedCache.java) block:RedisBackedCache
+    [Pre-Testcontainers test code](../examples/src/test/java/quickstart/RedisBackedCacheIntTestStep0.java) block:RedisBackedCacheIntTestStep0
```

Comment:
the new code uses `StatefulRedisConnection` while the old one is based on `RedisBackedCache`, makes the comparison not clear